scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/util/Ops.java

```diff
@@ -162,30 +162,41 @@ public static Class<?> findFunctionalInterface(Class<?> type) {
 	}
 
 	/**
-	 * Attempts to find the single functional method of the specified
-	 * class, by scanning the for functional interfaces. If there
-	 * is no functional interface, null will be returned.
+	 * Attempts to find the single functional method of the specified class, by
+	 * scanning the for functional interfaces. If there is no functional
+	 * interface, null will be returned.
 	 * 
-	 * @param cls
-	 * @return
+	 * @param cls the {@link Class} implmenting some {@link FunctionalInterface}
+	 * @return the functional {@link Method} of {@code cls}
+	 * @throws IllegalArgumentException if {@code cls} does not implement a
+	 *           {@link FunctionalInterface}
 	 */
 	public static Method findFunctionalMethod(Class<?> cls) {
+		Method m = fMethod(cls);
+		if (m == null) {
+			throw new IllegalArgumentException("Op type" + cls +
+				" does not have a functional method!");
+		}
+		return m;
+	}
+
+	private static Method fMethod(Class<?> cls) {
 		Class<?> iFace = findFunctionalInterface(cls);
 		if (iFace == null) {
 			return null;
 		}
-		
-		List<Method> nonDefaults = Arrays.stream(iFace.getMethods())
-				.filter(m -> !m.isDefault()).collect(Collectors.toList());
-		
+
+		List<Method> nonDefaults = Arrays.stream(iFace.getMethods()).filter(m -> !m
+			.isDefault()).collect(Collectors.toList());
```

Comment:
This change is the victim of autoformatting, which made the line wrapping worse.

Reply:
Fixed!